src/main/java/hudson/plugins/sonar/SonarInstallation.java

```diff
@@ -92,7 +96,8 @@ public SonarInstallation(
     @CheckForNull String mojoVersion,
     @CheckForNull String additionalProperties,
     @CheckForNull String additionalAnalysisProperties,
-    @CheckForNull TriggersConfig triggers) {
+    @CheckForNull TriggersConfig triggers,
+    boolean useTokenProperty) {
```

Comment:
by adding a new non nullable parameter to the constructor this is a breaking change for configuration as code users and you've updated the config files to make it pass.

This may be okay and just calling it out here in case.

If you wanted to make it compatible you can set it via a setter annotated with `@DataBoundSetter`